app/frontend/src/react-app-env.d.ts

```diff
@@ -6,8 +6,9 @@ declare namespace NodeJS {
     NODE_ENV: "development" | "production" | "test";
     REACT_APP_COUCHERS_ENV: "prod" | "preview" | "dev";
     REACT_APP_API_BASE_URL: string;
-    REACT_APP_IS_POST_BETA_ENABLED: string;
+    REACT_APP_IS_POST_BETA_ENABLED: "true" | undefined;
     REACT_APP_VERSION: string;
-    REACT_APP_IS_COMMUNITIES_ENABLED: string;
+    REACT_APP_IS_COMMUNITIES_ENABLED: "true" | undefined;
+    REACT_APP_IS_VERIFICATION_ENABLED: "true" | undefined;
```

Comment:
```suggestion
    REACT_APP_IS_COMMUNITIES_ENABLED?: "true";
    REACT_APP_IS_VERIFICATION_ENABLED?: "true";
```
I think this is basically the same for our use case and slightly shorter to write